src/mlpack/core/dists/regression_distribution.hpp

```diff
@@ -98,57 +99,58 @@ class RegressionDistribution
   void Train(const arma::mat& observations);
 
   /**
-   * Estimate parameters using provided observation weights
+   * Estimate parameters using provided observation weights.
    *
-   * @param weights probability that given observation is from distribution
+   * @param observations List of observations.
+   * @param weights Probability that given observation is from distribution.
    */
   mlpack_deprecated void Train(const arma::mat& observations,
                                const arma::vec& weights);
 
   /**
-   * Estimate parameters using provided observation weights
+   * Estimate parameters using provided observation weights.
    *
-   * @param weights probability that given observation is from distribution
+   * @param observations List of observations.
+   * @param weights Probability that given observation is from distribution.
    */
   void Train(const arma::mat& observations, const arma::rowvec& weights);
 
   /**
-  * Evaluate probability density function of given observation
-  *
-  * @param observation point to evaluate probability at
-  */
+   * Evaluate probability density function of given observation.
+   *
+   * @param observation Point to evaluate probability at.
+   */
   double Probability(const arma::vec& observation) const;
 
   /**
-  * Evaluate log probability density function of given observation
-  *
-  * @param observation point to evaluate log probability at
-  */
-  double LogProbability(const arma::vec& observation) const {
-    return log(Probability(observation));
-  }
+   * Evaluate log probability density function of given observation.
+   *
+   * @param observation Point to evaluate log probability at.
+   */
+  double LogProbability(const arma::vec& observation) const
+  { return log(Probability(observation)); }
```

Comment:
Personally I don't like the style, I would probably put it one another line, not sure the style is used at other places.

Reply:
Ah, thanks for pointing it out!! I'll edit it. :) 👍

Reply:
@KimSangYeon-DGU Yeah, your quoted style should be used.